pkg/controller/utils/certs.go

```diff
@@ -89,7 +88,7 @@ func EnsureCertificateSecret(secretName string, secret *corev1.Secret, keyName s
 }
 
 // Check if object is owned by the resource with given UID.
-func isOwnedByUID(obj client.Object, uid types.UID) bool {
+func IsOwnedByUID(obj client.Object, uid types.UID) bool {
```

Comment:
since it isn't needed now you should switch this back to not being exported (lowercase the Is)

Reply:
fixed